phylogenetic/defaults/auspice_config.json

```diff
@@ -68,13 +73,14 @@
   },
   "metadata_columns": [
     "strain",
+    "authors",
```

Comment:
_non-blocking_

If a field is included as one of the `colorings` or one of the `filters`, then it does not need to be included in `metadata_columns`.

Reply:
Thank you for the insight! I ran a check (to convince myself) and updated the commit accordingly. I'll aim at merging in a bit.